src/style.css

```diff
@@ -0,0 +1,102 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+.div {
+  display: block;
+}
+
+.search__form {
+  width: 100%;
+  display: flex;
+  height: 70px;
+  margin: 20px 0;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 3px;
+}
+
+.search__form-sm {
+  height: 42px;
+}
+
+.search__form-input {
+  font-family: inherit;
+  font-weight: inherit;
+  color: #3d4e61;
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+  height: 100%;
+  width: 100%;
+  border: 0;
+  border-radius: 3px;
+  /* cursor:pointer; */
+}
+
+.search__form-input-sm {
+  font-family: inherit;
+  font-weight: 300;
+  font-size: 14px;
+  padding-left: 33px;
+  line-height: 19px;
+}
+
+.search__form-input-sm::placeholder {
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.search__box {
+  width: 100%;
```

Comment:
an element with a display block automatically has a width of 100%

Reply:
This works correctly without display: block;

Reply:
this comment mean:
this element has display block by default, so has width 100% by default, so - it's redundant value

Reply:
This works correctly without display: block; but does not work correctly without width:100%;